pkg/codegen/codegen.go

```diff
@@ -294,7 +299,7 @@ func GenerateTypeDefinitions(t *template.Template, swagger *openapi3.T, ops []Op
 		return "", fmt.Errorf("error generating allOf boilerplate: %w", err)
 	}
 
-	typeDefinitions := strings.Join([]string{enumsOut, typesOut, paramTypesOut, allOfBoilerplate}, "")
+	typeDefinitions := strings.Join([]string{enumsOut, typesOut, enumTypesOut, paramTypesOut, allOfBoilerplate}, "")
```

Comment:
We can probably just use `+` here.

Reply:
yeah, would become a constant expression